### PR TITLE
feat: Aggregator processes LooksRare trades

### PR DESCRIPTION
**Pull request by 0xhiroshi**

### CODE REVIEW

File: subgraphs/aggregator/networks.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "mainnet": {
```
Comment (0xJurassicPunk) [inline subgraphs/aggregator/networks.json:2]: Let's add Goerli addresses here too

Comment (0xhiroshi) [reply subgraphs/aggregator/networks.json:2]: Since the `networks.json` is written to by The Graph, maybe we shouldn't manually change it?

Comment (0xJurassicPunk) [reply subgraphs/aggregator/networks.json:2]: That can be used as a config file or has its purpose changed?

See (for reference). https://github.com/LooksRare/subgraph/blob/master/subgraphs/looks-distribution/networks.json

**Review by 0xJurassicPunk — APPROVED** [review]

File: subgraphs/aggregator/networks.json (hunk 1 above)
Comment (0xhiroshi) [reply subgraphs/aggregator/networks.json:2]: Added

https://github.com/LooksRare/subgraph/pull/87/commits/4fef7a6ebb95b15aaf11eb8c575024aea6ef42bd

**Pull request merged**